skietbaan_fe/src/components/NavbarMenuUser.js

```diff
@@ -90,51 +86,34 @@ class NavbarMenuUser extends Component {
 			className="more-icon-grey" alt='More icon to expand tray'></img>
 	}
 
-	GoToLeaderboard (){
-		window.location = "/home";
+	GoTo (page){
+		window.location = page;
 	}
-
-	GoToScoreCapture(){
-		window.location = "/scorecapture";
-	}
-
-	GoToProfile(){
-		window.location = "/profile";
-	}
-
-	GoToDocuments(){
-		window.location = "/documents";
-	}
-
-	GoToNotifications(){
-		window.location = "/notifications";
-	}
-
 	render() {
 		// rendering the basic navbar within the render class
 		return (
 			<div >
 				<table className="navbar-admin">
 				<tbody>
 					<tr className="first-row-navbar">
-						<td className="columns" onClick={() => this.GoToLeaderboard()}>
+						<td className="columns" onClick={() => this.GoToLeaderboard("/home")}>
```

Comment:
You did not update the function call here

Reply:
done